lib/janus/session.ex

```diff
@@ -124,19 +128,22 @@ defmodule Janus.Session do
           event_manager = session.event_manager
           case data do
             %{janus: "keepalive"} -> GenEvent.notify(event_manager, {:keepalive})
+            %{janus: "event", plugindata: plugindata, sender: sender} ->
+              jsep = data[:jsep]
+              GenEvent.notify(event_manager, {:event, pid, data, jsep, sender})
```

Comment:
This is still including the sender. Is there any reason you need the explicit sender and can't match against `pid`, which would be unique per sender anyway?